internal/k8s/serialize_test.go

```diff
@@ -73,3 +73,32 @@ func TestHelmGeneratedRedisYAML(t *testing.T) {
 		"StatefulSet",
 	}, kinds)
 }
+
+func TestCRDYAML(t *testing.T) {
+	entities, err := ParseYAMLFromString(testyaml.CRDYAML)
+	assert.NoError(t, err)
```

Comment:
you should `if err != nil {t.Fatal(err)}` here so we don't continue to execute even if this step is borked and then get weird and confusing errors further down the test.

Reply:
i added a helper for it!